tests/Database/EmptyQueryTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rebing\GraphQL\Tests\Database;
+
+use Rebing\GraphQL\Tests\Support\Traits\SqlAssertionTrait;
+use Rebing\GraphQL\Tests\TestCaseDatabase;
+
+class EmptyQueryTest extends TestCaseDatabase
+{
+    use SqlAssertionTrait;
```

Comment:
Can be removed (and the `sqlCounterReset` call), as no test is really using this